Experiencing issues with building an `epicli` image. Reproduced locally and in the pipeline.
```
ERROR:  Error installing bcrypt_pbkdf:
	ERROR: Failed to build gem native extension.

    current directory: /var/lib/gems/2.7.0/gems/bcrypt_pbkdf-1.1.0/ext/mri
/usr/bin/ruby2.7 -I /usr/lib/ruby/vendor_ruby -r ./siteconf20211209-1156-okxfa7.rb extconf.rb
creating Makefile

current directory: /var/lib/gems/2.7.0/gems/bcrypt_pbkdf-1.1.0/ext/mri
make DESTDIR\= clean

current directory: /var/lib/gems/2.7.0/gems/bcrypt_pbkdf-1.1.0/ext/mri
make DESTDIR\=
compiling bcrypt_pbkdf.c
bcrypt_pbkdf.c:18:10: fatal error: sys/types.h: No such file or directory
   18 | #include <sys/types.h>
      |          ^~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:245: bcrypt_pbkdf.o] Error 1

make failed, exit code 2

Gem files will remain installed in /var/lib/gems/2.7.0/gems/bcrypt_pbkdf-1.1.0 for inspection.
Results logged to /var/lib/gems/2.7.0/extensions/x86_64-linux/2.7.0/bcrypt_pbkdf-1.1.0/gem_make.out
Building native extensions. This could take a while...

ERROR:  Error installing ed25519:
	ERROR: Failed to build gem native extension.

    current directory: /var/lib/gems/2.7.0/gems/ed25519-1.2.4/ext/ed25519_ref10
/usr/bin/ruby2.7 -I /usr/lib/ruby/vendor_ruby -r ./siteconf20211209-1156-js1gdp.rb extconf.rb
creating Makefile

current directory: /var/lib/gems/2.7.0/gems/ed25519-1.2.4/ext/ed25519_ref10
make DESTDIR\= clean

current directory: /var/lib/gems/2.7.0/gems/ed25519-1.2.4/ext/ed25519_ref10
make DESTDIR\=
compiling ed25519_ref10.c
In file included from /usr/include/ruby-2.7.0/ruby/ruby.h:29,
                 from /usr/include/ruby-2.7.0/ruby.h:33,
                 from ed25519_ref10.c:1:
/usr/include/ruby-2.7.0/ruby/defines.h:126:10: fatal error: stdio.h: No such file or directory
  126 | #include <stdio.h>
      |          ^~~~~~~~~
compilation terminated.
make: *** [Makefile:245: ed25519_ref10.o] Error 1

make failed, exit code 2

Gem files will remain installed in /var/lib/gems/2.7.0/gems/ed25519-1.2.4 for inspection.
Results logged to /var/lib/gems/2.7.0/extensions/x86_64-linux/2.7.0/ed25519-1.2.4/gem_make.out

```

--- a/Dockerfile
+++ b/Dockerfile
@@ -15,7 +15,7 @@ COPY . /epicli
 RUN : INSTALL APT REQUIREMENTS \
     && apt-get update \
     && apt-get install --no-install-recommends -y \
-        autossh curl gcc jq libcap2-bin libffi-dev make musl-dev openssh-client procps psmisc ruby-full sudo tar unzip vim \
+        autossh curl gcc jq libcap2-bin libc6-dev libffi-dev make musl-dev openssh-client procps psmisc ruby-full sudo tar unzip vim \
 \
     && : INSTALL HELM BINARY \
     && curl -fsSLO https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz \

